docs/running-rules.md

```diff
@@ -9,39 +9,39 @@ import MoreHelp from "/src/components/MoreHelp"
+- Local rules:
+  - Create and run one-off [ephemeral rules](#ephemeral-rules) in the command line.
+  - Create and run [YAML-defined rules in a file](#creating-and-using-yaml-defined-rules-file).
+- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
 
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
+You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI. For more information, see [Semgrep CI overview](semgrep-ci/overview.md).
 
-You can run existing and custom Semgrep rules locally with the Semrgep command line tool or continuously with Semgrep CI. See [Getting started](../getting-started/) for their respective installation and setup.
+## Running Semgrep Registry rules locally
 
-## Run Registry rules
-
-Explore the [Semgrep Registry](https://semgrep.dev/explore) and run rules and rulesets via:
-
-```sh
-# Automatically survey languages and frameworks and run recommended Registry rules
-semgrep --config=auto path/to/src
+Try to run the following example of [Semgrep Registry](https://semgrep.dev/explore) rules:
 
-# Run a ruleset with rules for many languages using --config
-semgrep --config=<ruleset-id> path/to/src
-
-# Run simultaneously with Registry rulesets and local rules
-semgrep --config=<ruleset-id> --config=path/to/yml path/to/src
-```
+1. Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code:
+    ```sh
+    semgrep --config=auto path/to/src
+    ```
+2. Explore the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
+3. On the page of the rule, click **Run Locally**.
+4. Copy the code for local install, and then add the path to the source code you want to check:
+    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
+5. Optional: Run registry rules simultaneously with local rules:
+   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/YML PATH/TO/SRC</span></code></pre>
 
 :::note
 * `--config auto` sends your repository's project URL to [Semgrep Registry](https://semgrep.dev/r) to find rules configured for your repository and as a key for cached rule recommendations.
 * When Semgrep Registry is used, [usage metrics](../metrics) are collected by default.
 :::
 
-Rulesets can be added to Semgrep CI scans using their "Add to Policy" button on Semgrep Community and Semgrep Team.
+Add rulesets to Semgrep in CI scans using their **Add to Policy** button on Semgrep Community and Semgrep Team tiers.
```

Comment:
Fairly certain this is outdated and superceded by "Add to Rule Board". If this is beyond the scope of this doc we can link to the docs for the rule board (suggestion only).

Reply:
It depends on how far along we think the user is at their journey. Some users are **prospecting a path** and some users are already decided on what they want to do. When using Semgrep, they need to know what their options are so they can also frame their goal.

Reply:
I would lead readers more to Semgrep App getting started... this seems like a less heavy solution...

Reply:
@s-santillan but... we can still include this, I just think that it is a bit of information overload to see all of these links in the docs. :D 

```markdown
:::info
Try Semgrep [Rule Board](semgrep-app/rule-board.md) to determine which rules Semgrep uses and what action Semgrep undertakes when it generates a finding.
:::
```